addons/mail/static/src/messaging/component/activity_mark_done_popover/activity_mark_done_popover.xml

```diff
@@ -7,16 +7,16 @@
                 <textarea class="form-control o_ActivityMarkDonePopover_feedback" rows="3" placeholder="Write Feedback" t-ref="feedbackTextarea"/>
                 <div class="o_ActivityMarkDonePopover_buttons">
                     <t t-if="!activity.forceNext">
-                        <button type="button" class="btn btn-sm btn-primary" t-on-click="_onClickDoneAndScheduleNext" t-esc="DONE_AND_SCHEDULE_NEXT" t-att-data-previous-activity-type-id="previous_activity_type_id"/>
-                        <button type="button" class="btn btn-sm btn-primary" t-on-click="_onClickDone">
+                        <button type="button" class="o_ActivityMarkDonePopover_doneScheduleNextButton btn btn-sm btn-primary" t-on-click="_onClickDoneAndScheduleNext" t-esc="DONE_AND_SCHEDULE_NEXT" t-att-data-previous-activity-type-id="previous_activity_type_id"/>
+                        <button type="button" class="o_ActivityMarkDonePopover_doneButton btn btn-sm btn-primary" t-on-click="_onClickDone">
                             Done
                         </button>
-                        <button type="button" class="btn btn-sm btn-link" t-on-click="_onClickDiscard">
+                        <button type="button" class="o_ActivityMarkDonePopover_discardButton btn btn-sm btn-link" t-on-click="_onClickDiscard">
                             Discard
                         </button>
                     </t>
                     <t t-else="">
-                        <button type="button" class="btn btn-sm btn-primary" t-esc="DONE_AND_SCHEDULE_NEXT"/>
+                        <button type="button" class="o_ActivityMarkDonePopover_doneScheduleNextButton btn btn-sm btn-primary" t-esc="DONE_AND_SCHEDULE_NEXT"/>
```

Comment:
This button doesn't have a handler or anything? Shouldn't the if/else be reorganized so we have only one "definition" of this button, the one that is correct?